modules/acs-requirements.adoc

```diff
@@ -28,12 +28,11 @@ For more information, see link:https://access.redhat.com/node/5822721[{product-t
 
 Cluster nodes minimum requirements:
 
-* *Architecture*: amd64, ppc64le, or s390x
+* *Architecture*: `amd64`, `ppc64le`, or `s390x`
 +
 [NOTE]
 ====
-For ppc64le, or s390x architectures, you can only install {product-title-short} secured cluster services on {ibm-power}, {ibm-z}, and {ibm-linuxone} clusters.
-Central is not supported at this time.
+For `ppc64le`, or `s390x` architectures, you can only install {product-title-short} secured cluster services on {ibm-power}, {ibm-z}, and {ibm-linuxone} clusters.
```

Comment:
This statement is not valid for 4.3.0. The statement was introduced in earlier releases when _only_ Secured Cluster was supported on P&Z, but not Central. In 4.3.0 both Central and Secured Cluster are supported.

You took out `Central is not supported at this time` which is correct, but the sentence **still** says that _only_ Secured Cluster is supported:

> you can _only_ install {product-title-short} secured cluster services

Instead, this can say something like "starting from ACS 4.3.0 both Central and Secured Cluster Services are supported on P and Z".